docs/DeveloperGuide.md

```diff
@@ -200,7 +200,7 @@ The delete student feature is implemented in the `DeleteCommand` class and facil
 
 Given below is an example usage scenario and how the delete student mechanism behaves at each step.
 
-Step 1: The user executes `deleteStud A1234567X` to add a student. The `AddressBookParser` class determines that the command called is `deleteStud`, and therefore creates a new `DeleteCommandParser` instance to parse the command.
+Step 1: The user executes `delete A1234567X` to add a student. The `AddressBookParser` class determines that the command called is `delete`, and therefore creates a new `DeleteCommandParser` instance to parse the command.
```

Comment:
I thought we no longer have a `AddressBookParser` class?

Reply:
Sorry for the oversight! I will fix it. Thank you for spotting it.